includes/header.inc.php

```diff
@@ -6,12 +6,12 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<title>Meeting Organiser</title>
 
-		<link href="../css/bootstrap.min.css" rel="stylesheet">
-		<link href="../css/font-awesome.min.css" rel="stylesheet">
-		<link href="../css/datepicker3.css" rel="stylesheet">
-		<link href="../css/bootstrap-table.css" rel="stylesheet">
-		<link href="../css/styles.css" rel="stylesheet">
-		<link href="../css/event-details.css" rel="stylesheet" >
```

Comment:
same as the footer file, 404 due to pointing to directory root instead of project root